src/python/pants/backend/explorer/server/BUILD

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+python_sources()
+
+# relocated_files(
```

Comment:
xx

Reply:
Ah, at least the `python_distribution` should probably stay, uncommented.. but I was unsure of the details about it..